### ISSUE TITLE
Error caused by i3 workspace name with more than 1 colon character

### ISSUE THREAD

**Issue by cjbassi** [jaagr/polybar#1513]
Running i3-gaps with a workspace named '1:2:3' causes polybar to repeatedly output:
```
error: Failed to parse contents (reason: Unrecognized token '3')
```
and while the bar loads fine, no modules are shown. I did some digging and found that at some point, polybar backslash escape colons in workspace names, but only for the first colon.

## To Reproduce
```dosini
[bar/main]
modules-left = i3

[module/i3]
type = internal/i3
```

## Environment:
`i3 -v`:
```
i3 version 4.16 (2018-11-04) © 2009 Michael Stapelberg and contributors
```
`polybar -vvv`:
```
polybar 3.2.0-26-g11d66ee

Features: +alsa +curl +i3 -mpd +network +pulseaudio +xkeyboard

X extensions: +randr (+monitors) -render -damage -sync -composite +xkb +xrm +xcursor

Build type: RelWithDebInfo
Compiler: /usr/lib/ccache/bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wall -Wextra -Werror -Wno-noexcept-type -O2 -pedantic -pedantic-errors
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```

**Comment by patrick96** [comment]
Duplicate of #984

**Comment by patrick96** [comment]
Thanks for reporting. I already fixed this issue in #1457, only waiting for it to get merged.
